packages/mechanic-template/functions/svgVideo/index.js

```diff
@@ -1,7 +1,9 @@
 export const handler = async (params, mechanic) => {
-  const r = params.width / 3;
+  const r = params.radius;
   let x = 0;
 
+  const template = document.createElement("template");
```

Comment:
I hadn't considered the use of `<template />` here. I think it would be nice if function authors have the ability to just return an SVG string, so let's discuss this too.